### PR TITLE
Add titles to admonitions

### PR DESCRIPTION
**Pull request by gabegma**
## Description:
Add titles to admonitions for clarity.

## Checklist:

You should check all boxes before the PR is ready. If a box does not apply, check it to acknowledge
it.

* [x] **PRE-COMMIT.** You ran pre-commit on all commits, or else, you
  ran `pre-commit run --all-files` at the end.
* [x] **FRONTEND TYPES.** Regenerate the front-ent types if you played with types and routes.
  Run `cd webapp && yarn types` while the back-end is running.
* [x] **USER CHANGES.** The changes are added to CHANGELOG.md and the documentation, if they impact
  our users.
* [x] **DEV CHANGES.**
    * Update the documentation if this PR changes how to develop/launch on the app.
    * Update the `README` files and our wiki for any big design decisions, if relevant.
    * Add unit tests, docstrings, typing and comments for complex sections.

### CODE REVIEW

File: docs/docs/development/dev-practices.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ We have different hooks which will run `mypy`, reorder imports, clean unnecessar
 - `mypy` errors will never prevent a commit, but it will print the errors so that you can fix
     them if you introduced a new one.
 
-!!! tip
+!!! info "How to temporarily disable pre-commits?"
```
Comment (lindsaydbrin) [inline docs/docs/development/dev-practices.md:29]:
```suggestion
!!! info "How to temporarily disable pre-commits"
```

File: docs/docs/development/dev-practices.md [hunk 2]
```diff
@@ -41,12 +41,12 @@ make test     # It will check for formatting issues, run mypy and all unit tests
 
 You can also only run specific tests, using `poetry run pytest {path_to_test.py}`.
 
-!!! tip
+!!! tip "Clean cache for routers test"
```
Comment (lindsaydbrin) [inline docs/docs/development/dev-practices.md:44]:
```suggestion
!!! tip "Clean cache for routers tests"
```
I think this is plural?

File: docs/docs/getting-started/c-run.md [hunk 3]
```diff
@@ -3,7 +3,7 @@
 This page guides you through the process of running the app on your data and pipelines, using
 Docker. Different dataset and text classification models can be supported in Azimuth.
 
-!!! info
+!!! info "Launch Azimuth with no pipeline, or with multiple ones"
```
Comment (lindsaydbrin) [inline docs/docs/getting-started/c-run.md:6]:
```suggestion
!!! info "Launch Azimuth with no pipeline, or with multiple pipelines"
```

File: docs/docs/key-concepts/index.md [hunk 4]
```diff
@@ -35,9 +35,7 @@ While smart tags are computed automatically and cannot be changed, proposed acti
 that can be added by the user to identify a proposed action that should be done on a specific data
 sample.
 
-!!! example
-
-    Examples of **proposed actions**:
+!!! example "Proposed actions examples"
```
Comment (lindsaydbrin) [inline docs/docs/key-concepts/index.md:38]:
```suggestion
!!! example "Proposed action examples"
```
Do you think this should be singular for the adjective?

File: docs/docs/user-guide/index.md [hunk 5]
```diff
@@ -26,7 +26,7 @@ The top banner contains useful information and links.
 
 ![Screenshot](../_static/images/dashboard/top-banner.png)
 
-!!! tip
+!!! tip "Don't miss out on the exploration space!"
```
Comment (lindsaydbrin) [inline docs/docs/user-guide/index.md:29]: <3

**Review by lindsaydbrin — APPROVED** [review]
LGTM and great idea!  This is super helpful for reading and quick understanding.

Do you want to add a title for the admonition at line 90 in `docs/docs/user-guide/index.md`, or leave it as is?  Maybe "No predictions for file-based configs" or "...without a model" or something like that?

File: docs/docs/key-concepts/saliency.md [hunk 6]
```diff
@@ -41,7 +41,7 @@ the [Utterances Table](../user-guide/exploration-space/utterance-details.md).
   <figcaption>Saliency map of a specific utterance.</figcaption>
 </figure>
 
-!!! example
+!!! example "Saliency example"
```
[the remark this answers is not in the document]
Comment (gabegma) [reply docs/docs/key-concepts/saliency.md:44]: Oh good catch!! When I updated the screenshot, I did not change the example. I'll fix it, thanks!

File: docs/docs/key-concepts/saliency.md [hunk 7]
```diff
@@ -41,7 +41,7 @@ the [Utterances Table](../user-guide/exploration-space/utterance-details.md).
...
 
     In this example, `assistance` is the word that contributes the most to the prediction
     `transfer`, followed by `moving`.
```
Comment (gabegma) [inline docs/docs/key-concepts/saliency.md:47]:
```suggestion
    In this example, `bill` is the word that contributes the most to the prediction `bill_balance`.
```

**Pull request merged**